apps/backend/src/models/base.py

```
@@ -0,0 +1,20 @@
+# Create the declarative base
+Base = declarative_base()
+
+
+class BaseModel(Base):
+    """Base model for all authentication database models."""
+    
+    __abstract__ = True
+    
+    id = Column(Integer, primary_key=True, index=True)
+    created_at = Column(DateTime, default=datetime.utcnow, nullable=False)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow, nullable=False)
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)